roles/edpm_nova/templates/02-nova-host-specific.conf.j2

```diff
@@ -0,0 +1,2 @@
+[DEFAULT]
+my_ip = {{ ctlplane_ip }}
```

Comment:
so... the main openquestion i have is should we introduce an ansible var to select what interface this should be or hardcode the control plane.

for context this IP will need to be included in certs and or know hosts files so we likely need a way to align this to one of the ansibel managed interface rather then just the default route.

Reply:
I will look into making the nova management network configurable in the role and default it to ctlplane

Reply:
For the sake of progress I suggest to merge this as is and get back to extend configuration later.